proguard-com.twitter.sdk.android.twitter.txt

```diff
@@ -0,0 +1,17 @@
+# Twitter Core proguard configuration
+# '-include' this file in your proguard config
+# Autogenerated file -- Do not modify
+
+# retrofit specific
+-dontwarn com.squareup.okhttp.**
+-dontwarn com.google.appengine.api.urlfetch.**
+-dontwarn rx.**
+-dontwarn retrofit.**
+-keepattributes Signature
+-keepattributes *Annotation*
+-keep class com.squareup.okhttp.** { *; }
+-keep interface com.squareup.okhttp.** { *; }
+-keep class retrofit.** { *; }
+-keepclasseswithmembers class * {
+    @retrofit.http.* <methods>;
+}
```

Comment:
Dang. All things considered, that's a pretty heavy sdk lol. totally worth using and I'm glad we're not stuck w/ twitter4j anymore but still